composer.json

```diff
@@ -15,10 +15,12 @@
         "php": ">=5.3.2"
     },
     "require-dev": {
-        "react/event-loop": "0.3.*@dev"
+        "react/event-loop": "0.3.*@dev",
+        "react/promise": "~2.2"
     },
     "suggest": {
-        "react/event-loop": "For using event-loop based scheduling."
+        "react/event-loop": "For using event-loop based scheduling.",
+        "react/promise": "For converting promises to and from observables."
```

Comment:
This is a conceptual issue; this PR uses React\Promise in its core now...while not always required it's in there. [This article](http://php-and-symfony.matthiasnoback.nl/2014/04/theres-no-such-thing-as-an-optional-dependency/) touches on the subject. Basically the library should work in all cases without having to load suggested libraries. 

I would recommend either requiring React\Promise or making a separate bridge package.